util/git/git.go

```diff
@@ -11,20 +12,59 @@ import (
 
 // NormalizeGitURL normalizes a git URL for lookup and storage
 func NormalizeGitURL(repo string) string {
-	repoURL, _ := url.Parse(repo)
+	repoURL, err := url.Parse(repo)
+	if err != nil {
+		return strings.ToLower(repo)
+	}
 	return repoURL.String()
 }
 
+// IsSshURL returns true is supplied URL is SSH URL
+func IsSshURL(url string) bool {
+	return strings.Index(url, "git@") == 0
```

Comment:
@alexmt The URL library for Golang has a [URL parser](https://golang.org/pkg/net/url/#ParseRequestURI) that returns [a URL struct](https://golang.org/pkg/net/url/#URL) where you can access the scheme, user, and other attributes.  Wondering if this might help make this function more robust.

Reply:
Also wondering if a unit test could be written for this method.

Reply:
Unfortunately, URL parser considers SSH URL invalid and returns error. Therefore I had to use `strings.Index` instead. Unit tests have been implemented.